tests/Unit/Helpers/PointwiseFunctions/AnalyticSolutions/Burgers/CheckBurgersSolution.hpp

```diff
@@ -34,41 +34,16 @@ void check_burgers_solution(const Solution& solution,
                             const std::vector<double>& times) noexcept {
   static_assert(evolution::is_analytic_solution_v<Solution>,
                 "Solution was not derived from AnalyticSolution");
-  // Check that different functions are consistent.
+  // Check that DataVector and double functions are consistent.
   const tnsr::I<DataVector, 1> positions_tnsr{{{positions}}};
   for (const double time : times) {
     const auto value = solution.u(positions_tnsr, time);
-    const auto time_deriv = solution.du_dt(positions_tnsr, time);
     CHECK(get<Burgers::Tags::U>(solution.variables(
               positions_tnsr, time, tmpl::list<Burgers::Tags::U>{})) == value);
-    CHECK(get<Tags::dt<Burgers::Tags::U>>(solution.variables(
-              positions_tnsr, time,
-              tmpl::list<Tags::dt<Burgers::Tags::U>>{})) ==
-          time_deriv);
     for (size_t point = 0; point < positions.size(); ++point) {
       const tnsr::I<double, 1> xp{{{positions[point]}}};
       const double up = get(value)[point];
-      const double dtup = get(time_deriv)[point];
       CHECK(get(solution.u(xp, time)) == approx(up));
-      CHECK(get(solution.du_dt(xp, time)) == approx(dtup));
-      // Check that the time derivative is the derivative of the
-      // value.
-      CHECK(numerical_derivative(
-                [&solution, &xp](const std::array<double, 1>& t) noexcept {
-                  return std::array<double, 1>{{get(solution.u(xp, t[0]))}};
-                },
-                std::array<double, 1>{{time}}, 0, 1e-4)[0] ==
-            approx.epsilon(1e-10)(dtup));
-      // Check that the Burgers equation is satisfied.
```

Comment:
This is a useful check, even without analytic derivatives.  Can you use a numerical derivative instead of deleting it?

Reply:
Presumably yes.

But this is a slippery slope. We already have a helper function that uses numerical derivatives to check PDE-compliance (in smooth regions) for the ValenciaDivClean analytic solutions. We could add a similar one for Burgers solutions. But probably the right thing to do is to generalize and check this with all analytic solutions...?

Should this PR be dependent on that generalization?

Reply:
A general implementation would be fine, but this check is why we opened a bug rather than just removing the functions when we did the other systems, so it should not be removed without a replacement.

Reply:
Sounds reasonable.